Tasks/WindowsMachineFileCopy/task.loc.json

```diff
@@ -12,8 +12,8 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 2,
-    "Minor": 0,
-    "Patch": 4
+    "Minor": 1,
```

Comment:
just update patch version only, if we are not sure what is behavior with updating the Minor version

Reply:
lets check with @kmkumaran